Spigot-API-Patches/0042-Added-PlayerFlowerPotManipulateEvent-for-manipulatin.patch

```diff
@@ -0,0 +1,90 @@
+From f6a3be20c88046ed457cde196e01e397888d89c9 Mon Sep 17 00:00:00 2001
+From: AlphaBlend <whizkid3000@hotmail.com>
+Date: Sun, 20 Nov 2016 15:37:47 -0800
+Subject: [PATCH] Added PlayerFlowerPotManipulateEvent for manipulating
+ contents of flowerpots
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/event/player/PlayerFlowerPotManipulateEvent.java b/src/main/java/com/destroystokyo/paper/event/player/PlayerFlowerPotManipulateEvent.java
+new file mode 100644
+index 0000000..848e601
+--- /dev/null
++++ b/src/main/java/com/destroystokyo/paper/event/player/PlayerFlowerPotManipulateEvent.java
+@@ -0,0 +1,74 @@
++package com.destroystokyo.paper.event.player;
++
++import javax.annotation.Nonnull;
++import org.bukkit.block.Block;
++import org.bukkit.entity.Player;
++import org.bukkit.event.Cancellable;
++import org.bukkit.event.HandlerList;
++import org.bukkit.event.player.PlayerEvent;
++import org.bukkit.inventory.ItemStack;
++
++/**
++ * Called when a player places or takes an item in a flowerpot
```

Comment:
Last I checked, the only way to take a plant out of a flower pot is to actually break the pot..